front/lib/resources/group_resource.ts

```diff
@@ -189,6 +199,44 @@ export class GroupResource extends BaseResource<GroupModel> {
     return new this(GroupModel, group.get());
   }
 
+  async addMember(
+    auth: Authenticator,
+    userId: string,
+    transaction?: Transaction
+  ): Promise<void> {
+    const owner = auth.getNonNullableWorkspace();
+    const user = await UserResource.fetchById(userId);
+
+    if (!user) {
+      throw new Error("User not found.");
+    }
+    if (this.type !== "regular") {
+      throw new Error("Cannot add members to non-regular groups.");
+    }
+
+    const workspace = renderLightWorkspaceType({ workspace: owner });
```

Comment:
Do we really need this vs passing `owner` directly?

Reply:
Nope, changed it!